src/auth/auth-api-request.ts

```diff
@@ -811,6 +811,21 @@ export abstract class AbstractAuthRequestHandler {
     return this.invokeRequestHandler(this.getAuthUrlBuilder(), FIREBASE_AUTH_GET_ACCOUNT_INFO, request);
   }
 
+  public getAccountInfoByFederatedUid(federatedId: string, federatedUid: string): Promise<object> {
```

Comment:
Nit: The 2 args are only off by one character. Can we make them a bit more distinct?

Reply:
These are meant to match the proto parameters, so we can easily change them to more closely reflect that. Done.

Note that federatedId now becomes providerId... which sounds slightly looser, but I think is fine since the method itself has 'federated' right there in the name.